docs/usage/configuration/config_documentation.md

```diff
@@ -23,6 +23,14 @@ followed by a letter. Letters have the following meanings:
 For example, setting `redaction_retention_period: 5m` would remove redacted
 messages from the database after 5 minutes, rather than 5 months.
 
+In addition, configuration options referring to size use the following suffixes:
+
+* `M` = MB, or 1048576 bytes
+* `K` = KB, or 1024 bytes 
```

Comment:
1024 bytes is actually [a Kibibyte](https://en.wikipedia.org/wiki/Byte#Multiple-byte_units), which is abbreviated to KiB. Same for MiB!

<sub>It's also a very fun word to say.</sub>

Reply:
It is indeed :)